mlrun/feature_store/ingestion.py

```diff
@@ -243,6 +245,23 @@ def add_source_trigger(source, function):
             source.attributes["seek_to"],
             source.attributes["shards"],
         )
+    if isinstance(source, KafkaSource):
```

Comment:
@dinal why not add some method to `KafkaSource` (and v3io), which returns/adds the nuclio trigger, this way the ingestion is clean and we can add more triggers in a simpler way ..